pkgs/development/tools/build-managers/meson/default.nix

```diff
@@ -1,14 +1,16 @@
-{ lib, python3Packages }:
+{ lib, python3Packages, ninja }:
 python3Packages.buildPythonApplication rec {
-  version = "0.40.0";
+  version = "0.41.2";
   pname = "meson";
   name = "${pname}-${version}";
 
   src = python3Packages.fetchPypi {
     inherit pname version;
-    sha256 = "1hb6y5phzd5738rlpz78w8hfzk7sbxj81551mb7bbkkqz8ql1gjw";
+    sha256 = "0p69hir68ar3nzrjn0zjsnyzq181b0kq6arrcmxqpzl7g5qhf5xd";
   };
 
+  propagatedBuildInputs = [ ninja ];
```

Comment:
Why do you add `ninja`? I know `ninja` is typically used to eventually build the code, but `meson` works fine without `ninja`, right?